test/command_line/test_integrate.py

```diff
@@ -254,6 +254,34 @@ def test_integration_with_sample_size(dials_data, tmpdir):
     assert dict(table.experiment_identifiers()) == {0: "foo"}
 
 
+def test_basic_integration_with_profile_fitting(dials_data, tmpdir):
+
+    expts = dials_data("centroid_test_data") / "indexed.expt"
+    refls = dials_data("centroid_test_data") / "indexed.refl"
+
+    result = procrunner.run(
+        [
+            "dials.integrate",
+            "nproc=1",
+            expts,
+            refls,
+            "profile.fitting=True",
+            "sampling.integrate_all_reflections=False",
+            "sampling.minimum_sample_size=500",
+            "prediction.padding=0",
+        ],
+        working_directory=tmpdir,
+    )
+    assert not result.returncode and not result.stderr
+    table = flex.reflection_table.from_file(tmpdir / "integrated.refl")
```

Comment:
Propose stronger test would be to check that there are no reflection centroids in tile join regions?